src/Apps/Order/Components/ProcessingPayment.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react"
+import styled from "styled-components"
+import { Flex, Spinner, Text } from "@artsy/palette"
+
+export const ProcessingPayment: FC = () => (
```

Comment:
Same here, no implicit returns pls 🙏 It looks and feels tidy, but it always ends up a maintainence burden over time